src/main/java/org/humancellatlas/ingest/biomaterial/Biomaterial.java

```diff
@@ -64,4 +67,9 @@ public Biomaterial addAsDerivedByProcess(Process process) {
         return this;
     }
 
+    @JsonProperty(access=READ_ONLY)
+    public boolean isLinked() {
+        return !inputToProcesses.isEmpty() || !derivedByProcesses.isEmpty();
+    }
+
```

Comment:
@rdgoite could you check if this is not undoing the lazy loading? This might cause a performance issue

cc @rolando-ebi

Reply:
It looks like it will force the `inputToProcesses` and `derivedByProcesses` to be eagerly loaded when Biomaterial resources are read through the REST API. API users fetching a biomaterial resource would involve 3 database accesses as opposed to 1. This might not have a significant impact on performance but would be worth confirming and then assessing. 

Could also consider making a `../biomaterial/<id>/isLinked` link from Biomaterial resources to allow API users to deliberately get this info only when they need it.